encord/objects/ontology_labels_impl.py

```diff
@@ -1849,9 +1892,9 @@ def _parse_labels_from_dict(self, label_row_dict: dict):
             elif data_type == DataType.MISSING_DATA_TYPE:
                 raise NotImplementedError(f"Got an unexpected data type `{data_type}`")
 
+            # In the future, PDF and Text should come here
             elif data_type == DataType.AUDIO:
```

Comment:
Can we make a list: "NonGeometricDataTypes" and have it contain DataType.AUDIO etc rather than special casing AUDIO. All of this branching makes me squirm.

Reply:
hm...if I'm not mistaken, for the `exhaustive_guard` to work, we can't do that.

We'll have to define each `data_type` individually in the `if` statement
```
elif data_type == DataType.AUDIO or data_type == DataType.PDF

etc...
```

Reply:
If that's the case, then it's time to leave Python lol. Although I guess the LSP can't know that we haven't mutatewd NonGeometricDataTypes. I wonder if you labelled it as **Final** would work here. To indicate that it's a 'compile time const'. (Someone with more experience with C derivatives than me would know the exact word)